docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+1. Native histogram bucket boundaries may change (widen) dynamically if observations result in too many buckets. For more detail see [Limiting the number of buckets](#limiting-the-number-of-buckets).
+1. Native histogram bucket counters only count observations inside the bucket boundaries, whereas the classic histogram buckets only had an upper bound called `le` and counted all observations in the bucket and all lower buckets (cumulative).
+1. An instance of a native histogram metric only requires a single time series, because the buckets, sum of observations, and count of observations are stored in a single new data type - and not in separate time series using the float data type. Thus there are no `<metric>_bucket`, `<metric>_sum`, `<metric>_count` series, only `<metric>`.
+1. Querying native histograms via the Prometheus query language (PromQL) uses a different syntax. There are also a couple of new [functions](https://prometheus.io/docs/prometheus/latest/querying/functions/).
+
+For an introduction to native histograms, watch the following presentation: [Native Histograms in Prometheus](https://www.youtube.com/watch?v=AcmABV6NCYk).
+
+## Pros and cons
+
+This section describes the pros and cons of using native histograms compared to the classic Prometheus histograms. For more information and real example, see [Prometheus Native Histograms in Production](https://www.youtube.com/watch?v=TgINvIK9SYc&t=127s) video.
+
+### Pros
+
+1. Simpler instrumentation. No need to think about bucket boundaries, they are automatic.
+1. Better resolution in practice as custom bucket layouts are usually not that high resolution.
+1. Native histograms are compatible with each other due to the automatic layout, thus they can be easily combined. Caveat: the operation might scale down an operand to lower resolution to match the other operand.
+
+### Cons
+
+1. Observations might be distributed in a way that is not a good fit for the exponential bucket schema, e.g. sound pressure measured in decibels, which are already logarithmic on their own.
+1. If converting from an externally represented histogram with specific bucket boundaries, there is generally no precise match with the bucket boundaries of the native histogram and interpolation has to be applied.
+1. There is no way to set an arbitrary bucket boundary, e.g. one that is particularly interesting for an SLO definition. Ratios of observations above or below a given threshold generally have to be estimated by interpolation, rather than being precise as it is the case for a classic histogram with a configured bucket boundary at the given threshold.
+
+All three problems are mitigated by high resolution, which native histograms can provide at a much lower resource costs compared to classic histograms.
+
+## Use native histograms
```

Comment:
This is an extraneous section. Can it be removed?

Reply:
Stopped here for now because I need to hop back to other work.

Reply:
> This is an extraneous section. Can it be removed?

I meant line 46.

Reply:
It's meant to lead into the instrumentation bit, I'll add some text to it